kogito-codegen-modules/kogito-codegen-decisions/src/main/resources/class-templates/DecisionContainerQuarkusTemplate.java

```diff
@@ -16,6 +16,7 @@
 package $Package$;
 
 @javax.enterprise.context.ApplicationScoped()
+@io.quarkus.runtime.Startup()
```

Comment:
What about add a comment to motivate this change and prevent undesired future changes. Something like
```suggestion
// Required to force full reload during hot reload
@io.quarkus.runtime.Startup()
```

Reply:
done.